_adr/83/index.adoc

```diff
@@ -0,0 +1,131 @@
+---
+num: 0 # allocate an id when the draft is created
+title: Permission API
+status: "Draft"
+authors:
+  - "Massimiliano Ziccardi"
+tags:
+  - "kafka"
+#applies_padrs: # What PADRs does this ADR apply?
+#applies_patterns: # What APs does this ADR apply?
+---
+## Context and Problem Statement
```

Comment:
In a context of an eventual move to a KCP hosted API, how is that problem resolved? Is there such API? CC @pmuir

Reply:
Yes, kubernetes provides a SelfSubjectAccessReview API which is wrapped by `kubectl auth can-i` - which KCP supports - see https://kubernetes.io/docs/reference/access-authn-authz/authorization/#checking-api-access

Reply:
https://docs.openshift.com/container-platform/4.6/rest_api/authorization_apis/selfsubjectaccessreview-authorization-k8s-io-v1.html